src/pages/settings/Profile/ProfilePage.js

```diff
@@ -148,8 +148,8 @@ class ProfilePage extends Component {
                         onClose={PersonalDetails.clearAvatarErrors}
                     >
                         <AvatarWithImagePicker
-                            isUsingDefaultAvatar={lodashGet(currentUserDetails, 'avatar', '').includes('/images/avatars/avatar')}
-                            avatarURL={currentUserDetails.avatar}
+                            isUsingDefaultAvatar={ReportUtils.isDefaultAvatar(lodashGet(currentUserDetails, 'avatar', ''))}
+                            source={ReportUtils.getAvatar(currentUserDetails.avatar, currentUserDetails.login)}
```

Comment:
I feel like we should either use `lodashGet` everywhere in these 2 lines or nowhere

Reply:
Agreed!